Apps/Src/Voltage.c

```diff
@@ -194,6 +194,9 @@ SafetyStatus Voltage_OpenWire(void){
 
 	for(int32_t i = 0; i < NUM_MINIONS; i++) {
 		if(Minions[i].system_open_wire != 0){
+			if ((i == NUM_MINIONS -1) && ((Minions[i].system_open_wire & 0xEF) != 0)) { //The last Voltage board is only connected to 7 modules
```

Comment:
Could we make this dependent on the configuration?

Reply:
I think we could do that by adding more defines to the config file (like batteries missing in last board or something like that).

Reply:
Referenced this in #416